### PR TITLE
Revert "Merge branch 'master' into release_5.3.5.1"

### PR DESCRIPTION
**Pull request by tst-mbiancalana**
This reverts commit c9645fb59662959cf6afc9e30427119cb8b30b4f, reversing
changes made to 1dd99dccbf18c00f469cf4deb33530d7064ef1c6.

### CODE REVIEW

**Comment by tst-nfarah** [comment]
ci build failed.

**Comment by tst-mbiancalana** [comment]
I see that. it's because apt is failing to connect to the package repos.

File: README.md [hunk 1]
```diff
@@ -19,11 +19,6 @@ The application suite consists of three core components:
 
 ## Releases
 
-[Release 5.3.5](https://github.com/google/earthenterprise/releases/tag/5.3.5-1610.20)
```
Comment (tst-nfarah) [inline README.md:22]: 5.3.5.1 did not have these before ?

Comment (tst-mbiancalana) [reply README.md:22]: No, it didn't. It appears we need to get some updates from `release_5.3.5.0` that happened after this branch was made.

**Comment by tst-eclamar** [comment]
Looks good

**Pull request merged**